dashboard/db/migrate/20160818232419_add_url_param_to_stages.rb

```diff
@@ -0,0 +1,35 @@
+class AddUrlParamToStages < ActiveRecord::Migration
+  def up
+    add_column :stages, :url_param, :string
+
+    Stage.reset_column_information
+    # Go through each script, and calculate url_param for lockable/unlockable
+    # stages
+    Script.all.each do |script|
```

Comment:
I can see if `rake seed` will repopulate, but is it safe to assume that `rake seed` will be run after a db:migrate? It would also mean we can't require non-null (or we need to provide some sort of default value)